### DISCUSSION TITLE
How to support URLs directly without using `remark-gfm`

### DISCUSSION THREAD

**Discussion by idontknowjava** [remarkjs/remark#1054] [Q&A]
I'm aware that [react-gfm](https://github.com/remarkjs/remark-gfm) allows the support of URLs directly within our markdown. But along with URLs, it also supports `strikethrough, tables, tasklists` which I don't want to support in an usecase.

I only want to support URLs functionality from `remark-gfm`, is there any way to achieve it?

Thanks in advance!

**Comment by wooorm** [comment]
Presumably, you mean `remark-gfm`.

I strongly recommend you use:
- existing markdown links: `<https://example.com>` (autolinks) or `[text](https://example.com)` (links with references)
- common markdown extensions, such as `remark-gfm`, all of it.

I strongly discourage you from making custom flavors of markdown.
If you really want to, you can pick and choose which extensions you want, by looking at how `remark-gfm` is made.
